core/src/main/scala/commandcenter/util/GraphicsUtil.scala

```diff
@@ -0,0 +1,9 @@
+package commandcenter.util
+
+import java.awt.{ GraphicsDevice, GraphicsEnvironment }
+
+object GraphicsUtil {
+  def isOpacitySupported: Boolean =
+    GraphicsEnvironment.getLocalGraphicsEnvironment.getDefaultScreenDevice
+      .isWindowTranslucencySupported(GraphicsDevice.WindowTranslucency.TRANSLUCENT)
```

Comment:
Since this isn't a pure function and there are times it can throw an exception (like in an headless environment), I would wrap this in `Task`.

Reply:
Fixed by using `Task`, and fallback to `false` if the check fails.